### DISCUSSION TITLE
Ampache 5 Subsonic problems

### DISCUSSION THREAD

**Discussion by Rodol66** [ampache/ampache#3096] [General]
In this moment i have 2 Ampache instalations running in the same server, one with 4 version and other with the las one, both whit same catalogs and same files source. The old one runs perfectly, but have some Subsonic problems with 5

I'm using Dsub in android and Volumio with Volusonic plugin at home, and have the same issue in both. When I configure the server only can see 5 folders of 5 groups with all his cds inside (i have more than 100) in one catalog, no one in the other 2, i don't know what's the difference with de rest.

I can solve this only in Dsub using "explore by tags" option, then i can see all my music.

Any idea? Regards

**Comment by lachlan-00** [comment]
do you have private catalogs set up?

in dsub under library -> select folder you should be able to select all folders and get everything

**Reply by Rodol66** [reply]
No private catalogs, and config file is 100% original. Is the same with all users, admins or regular ones. Seems to be a problem with artist list, beause the only five i can see with tags exploration deactivated have all albuns inside.

In Web player i can see all with all users

**Reply by lachlan-00** [reply]
do you have multiple catalogs listed as folders like this?
![image](https://user-images.githubusercontent.com/1305249/142689746-9c45b0bb-e09b-4372-8737-2210fc0f675f.png)

if you have only one catalog there might be an issues with single catalogs i haven't seen yet

**Comment by Rodol66** [comment]
I had 3catalogs, have deleted 2 of them but the problem is not solved. I cant see the difference between the 6 artist i can see and the others.

**Comment by Rodol66** [comment]
Same server, same music files, same configuration and no way. I will stay in Ampache 4 while it works and if there is no solution in later versions of 5 I will have to look for alternatives. Thank you so much

Ampache 4 server
![Screenshot_20211124-165110_DSub](https://user-images.githubusercontent.com/94574690/143272520-5f80e8d8-09b2-4146-be4a-05e19b09366e.jpg)

and with Ampache 5



![Screenshot_20211124-165056_DSub](https://user-images.githubusercontent.com/94574690/143272574-70e16a6d-3ae8-4fd1-9b47-0f37c44bfa8c.jpg)
